plasma_framework/test/src/exits/payment/PaymentStartInFlightExit.test.js

```diff
@@ -429,11 +431,24 @@ contract('PaymentInFlightExitRouter', ([_, alice, bob, carol]) => {
 
                 await expectRevert(
                     this.exitGame.startInFlightExit(args, { from: alice, value: IN_FLIGHT_EXIT_BOND }),
-                    'Invalid transaction, spends more than provided in inputs',
+                    'In-flight transaction must have unique inputs',
                 );
             });
+        });
 
-            it('should fail when in-flight transaction with multiple tokens inputs/outputs overspends', async () => {
+        describe('when in-flight exit start is called', () => {
+            beforeEach(async () => {
+                this.framework = await SpyPlasmaFramework.new(
+                    MIN_EXIT_PERIOD, DUMMY_INITIAL_IMMUNE_VAULTS_NUM, INITIAL_IMMUNE_EXIT_GAME_NUM,
+                );
+                this.exitGame = await PaymentInFlightExitRouter.new(
+                    this.framework.address,
+                    this.spendingConditionRegistry.address,
+                    this.stateTransitionVerifierReject.address,
+                );
+            });
+
+            it('should fail when in-flight transaction is an invalid state transistion', async () => {
```

Comment:
```suggestion
            it('should fail when the in-flight transaction has an invalid state transistion', async () => {
```

Reply:
I might try to test both verifier returns false and reverts. I think both are potential results in such verifier.

Reply:
done